.gitignore

```diff
@@ -66,13 +55,11 @@ static/src/stylesheets/icons/*
 
 # Eclipse #
 .project
-.classpath
 .settings/
 .metadata
 .classpath
 .springBeans
 .cache
-bower_components
```

Comment:
sadly we do - the bower installation process actually copies files from your local `bower_components` to `components` to be checked in, for long-forgotten reasons. it's on my list to remove bower once we're in webpack land, but for the moment, anyone who updates the bower deps will have this folder locally...